### DISCUSSION TITLE
a lot of errormasseges in the log file Register 59 and 219

### DISCUSSION THREAD

**Discussion by concierge-at** [kellerza/sunsynk#308] [Q&A] [answered]
Hello, I haven't been here long, and homeassistant is very powerful, the integration of Solarman Yellow with mqtt works very well. Today I looked through the logs and there were a lot of errors

`
00:05:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
00:13:43 ERROR   Error retry attempt(0): Connection already closed
00:15:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
00:20:28 ERROR   Error retry attempt(0): Connection already closed
00:25:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
00:27:12 ERROR   Error retry attempt(0): Connection already closed
00:30:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
00:38:28 ERROR   Error retry attempt(0): Connection already closed
00:40:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
00:40:48 ERROR   Error retry attempt(0): Connection already closed
00:45:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
01:07:32 ERROR   Error retry attempt(0): Connection already closed
01:10:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
02:07:37 ERROR   Error retry attempt(0): Connection already closed
02:10:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
02:10:03 ERROR   Error retry attempt(0): Connection already closed
02:15:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
02:45:28 ERROR   Error retry attempt(0): Connection already closed
02:50:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
02:53:18 ERROR   Error retry attempt(0): Connection already closed
02:55:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
03:43:23 ERROR   Error retry attempt(0): Connection already closed
03:45:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
03:46:03 ERROR   Error retry attempt(0): Connection already closed
03:48:18 ERROR   Error retry attempt(0): Connection already closed
03:50:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
03:50:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
05:08:18 ERROR   Error retry attempt(0): Connection already closed
05:10:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
05:10:33 ERROR   Error retry attempt(0): Connection already closed
05:15:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
05:28:58 ERROR   Error retry attempt(0): Connection already closed
05:30:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
05:35:43 ERROR   Error retry attempt(0): Connection already closed
05:40:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
05:42:23 ERROR   Error retry attempt(0): Connection already closed
05:44:48 ERROR   Error retry attempt(0): Connection already closed
05:45:00 ERROR   (2 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
05:51:43 ERROR   Error retry attempt(0): Connection already closed
05:55:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
05:58:28 ERROR   Error retry attempt(0): Connection already closed
06:00:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
06:57:47 ERROR   Error retry attempt(0): Connection already closed
07:00:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
07:00:03 ERROR   Error retry attempt(0): Connection already closed
07:02:33 ERROR   Error retry attempt(0): Connection already closed
07:04:48 ERROR   Error retry attempt(0): Connection already closed
07:05:00 ERROR   (2 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
07:05:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
07:07:18 ERROR   Error retry attempt(0): Connection already closed
07:10:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
08:20:18 ERROR   Error retry attempt(0): Connection already closed
08:25:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
08:29:48 ERROR   Error retry attempt(0): Connection already closed
08:30:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
08:36:27 ERROR   Error retry attempt(0): Connection already closed
08:40:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
08:43:17 ERROR   Error retry attempt(0): Connection already closed
08:45:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
08:58:03 ERROR   Error retry attempt(0): Connection already closed
09:00:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
09:04:48 ERROR   Error retry attempt(0): Connection already closed
09:05:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
09:07:13 ERROR   Error retry attempt(0): Connection already closed
09:10:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
09:13:58 ERROR   Error retry attempt(0): Connection already closed
09:15:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
09:20:38 ERROR   Error retry attempt(0): Connection already closed
09:25:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
09:40:33 ERROR   Error retry attempt(0): Connection already closed
09:42:58 ERROR   Error retry attempt(0): Connection already closed
09:45:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
09:45:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
10:52:43 ERROR   Error retry attempt(0): Connection already closed
10:55:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
10:55:02 ERROR   Error retry attempt(0): Connection already closed
11:00:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
11:01:49 ERROR   Error retry attempt(0): Connection already closed
11:05:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
11:18:13 ERROR   Error retry attempt(0): Connection already closed
11:20:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
11:20:32 ERROR   Error retry attempt(0): Connection already closed
11:25:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
11:27:12 ERROR   Error retry attempt(0): Connection already closed
11:29:32 ERROR   Error retry attempt(0): Connection already closed
11:30:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
11:30:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
11:57:21 ERROR   Error retry attempt(0): Connection already closed
11:59:41 ERROR   Error retry attempt(0): Connection already closed
12:00:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
12:00:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
13:43:27 ERROR   Error retry attempt(0): Connection already closed
13:45:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
14:52:32 ERROR   Error retry attempt(0): Connection already closed
14:54:48 ERROR   Error retry attempt(0): Connection already closed
14:55:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
14:55:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 219 poll_need_to_read
15:09:48 ERROR   Error retry attempt(0): Connection already closed
15:10:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
15:27:48 ERROR   Error retry attempt(0): Connection already closed
15:30:00 ERROR   (1 in 5 min) OSError in read_ss: timeout reading 1 registers from 59 poll_need_to_read
`

have I done something wrong here? Or can it be ignored or is there a solution I could do. 
Thanks in advance and thanks again for the integration
conci

**Comment by kellerza** [comment, marked as answer]
#312
